.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           npm run build
           npm run build:types
       - name: Test
-        run: npm t -- --coverage --colors
+        run: npm t -- --coverage --colors --randomize --showSeed
```

Comment:
Could set these in the config file, and not only in CI, wdyt?
https://jestjs.io/docs/configuration#randomize-boolean
https://jestjs.io/docs/configuration#showseed-boolean